pycontext/context.py

```diff
@@ -189,12 +221,15 @@ def __deepcopy__(self, memo=None):
         # that ensures that copy of correct subclass is created
         new_context = self.__class__()
 
-        # need to loop over the vars to copy all
-        # class attributes including possibly added by subclasses
```

Comment:
subclasses can potentially add additional slots.maybe a good idea to deepcopy them as well if they are present? thoughts